library/core/src/main/java/com/google/android/exoplayer2/text/ttml/TtmlDecoder.java

```diff
@@ -127,7 +130,7 @@ protected TtmlSubtitle decode(byte[] bytes, int length, boolean reset)
               Log.i(TAG, "Ignoring unsupported tag: " + xmlParser.getName());
               unsupportedNodeDepth++;
             } else if (TtmlNode.TAG_HEAD.equals(name)) {
-              parseHeader(xmlParser, globalStyles, regionMap, cellResolution);
+              parseHeader(xmlParser, globalStyles, regionMap, cellResolution, imageMap);
```

Comment:
nit - It might be nice for the two "Map" variables to be next to one another (consider this ignorable :))